semgrep/semgrep/config_resolver.py

```diff
@@ -61,26 +64,165 @@
 }
 
 
+class ConfigPath:
+    def __init__(self, config_str: str) -> None:
+        """ returns the registry url for the config_str, if it is a registry entry """
```

Comment:
@emjin Update this comment

Reply:
Must note that this updates `metric_manager.using_server`

Reply:
Seems like these changes are in conflict with changes in https://github.com/returntocorp/semgrep/pull/3944/files#diff-fa93cdfb09f1a1e0580936c43d7b3fd0bcec220bc1e8163bcd2b54747ec159b5R214

Reply:
Is it just the name?

Reply:
Let's prioritize this PR first.